backend/corpora/common/entities/dataset.py

```diff
@@ -96,6 +96,15 @@ def get(cls, session: Session, dataset_uuid, include_tombstones=False) -> "Datas
                 return None
         return dataset
 
+    @classmethod
+    def get_dataset_from_curator_tag(cls, session: Session, collection_id, curator_tag) -> "Dataset":
+        dataset = (
+            session.query(cls.table)
+            .filter(cls.table.collection_id == collection_id, cls.table.curator_tag == curator_tag)
```

Comment:
This is what I _should_ have done for the [dataset submission lambda](https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/backend/corpora/dataset_submissions/app.py#L103). 🤦